back-end/app/Http/Controllers/User/UsersController.php

```diff
@@ -72,9 +72,9 @@ public function update(Request $request, $user_id)
             'cover_url' => 'string|max:999|nullable',
```

Comment:
```suggestion
            'cover_url' => 'url|max:999|nullable',
```

Reply:
Great! Thank you Sir Chev for that, I'll fix it right now.